src/main/scala/com/github/nikalaikina/poehali/Boot.scala

```diff
@@ -16,44 +17,42 @@ import scalacache.ScalaCache
 import scalacache.guava.GuavaCache
 
 object Boot extends App {
-  val host = "localhost"
+  val host = "0.0.0.0"
   val port = 8888
 
+  import com.github.nikalaikina.poehali.util.TimeoutImplicits.waitForever
```

Comment:
`import com.github.nikalaikina.poehali.util.TimeoutImplicits` before object definition

in object body 

`import TimeoutImplicits.waitForever`